README.md

```diff
@@ -9,19 +9,19 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # CXX-Qt
 
-CXX-Qt is a library that automatically generates code to transfer data between Rust and C++ through common interfaces
-such as QObjects that can be exposed directly into QML. It relies on the cxx crate internally to achieve this and thus
-it is recommended that any interactions with Qt that are not covered by the built-in code generators should be done
-directly in C++ and connected to relevant Rust logic by writing additional cxx code. The CXX-Qt build system is based
-on CMake, but is compatible with cxx on its own as well.
-
-The examples folder contains an example application using the CXX-Qt crate and will be used for development and testing
-purposes. The cxx-qt folder contains the source for the actual crate which contains a proc-macro. The cxx-qt-gen folder
-contains the source for a crate which extracts and generates C++ and Rust source code. The cxx-qt-build folder contains
-the source for a crate which provides helper functions to be used in a `build.rs` file.
-
-Initially the projects in the examples folder will also serve as a template for new projects should use CXX-Qt.
-In future we might improve upon this with a custom CMake module for instance.
+CXX-Qt is a set of Rust crates for creating bidirectional Rust ⇄ C++ bindings with [Qt](https://www.qt.io/).
+It can be used to integrate Rust into C++ applications using CMake or used to build Rust applications with Cargo.
+CXX-Qt provides tools for implementing [QObject](https://doc.qt.io/qt-6/object.html) subclasses in Rust which can
+be used from C++, QML, and JavaScript. It consists of two parts:
+
+1. cxx-qt-lib, a library of Rust bindings to common QtCore and QtGui classes made with [CXX](https://cxx.rs/)
```

Comment:
personally prefer to use bullet points rather than numbers points if they aren't being referred to